src/QueueRegistry.cpp

```diff
@@ -11,21 +11,18 @@
 #include "appfwk/QueueRegistry.hpp"
 
 #include <map>
+#include <memory>
 #include <string>
 
 namespace dunedaq::appfwk {
 
-QueueRegistry* QueueRegistry::me_ = nullptr;
-
-QueueRegistry::QueueRegistry() {}
-
-QueueRegistry::~QueueRegistry() {}
+  std::unique_ptr<QueueRegistry> QueueRegistry::me_ = nullptr;
 
 QueueRegistry&
 QueueRegistry::get()
 {
   if (!me_) {
-    me_ = new QueueRegistry();
+    me_.reset( new QueueRegistry() );
```

Comment:
One of the "modernize" guidelines recommends using std::make_unique in this situation.

Reply:
Except that it doesn't build. Rescinding comment.